libs/sdk-ui-kit/src/Dialog/EmbedInsightDialog/index.ts

```diff
@@ -0,0 +1,27 @@
+// (C) 2022 GoodData Corporation
+
+export type {
+    IOptionsByDefinition,
+    InsightCodeType,
+    CodeLanguageType,
+    CodeOptionType,
+    IOptionsByReference,
+    UnitsType,
+} from "./EmbedInsightDialogBase/types";
+
+export { getDefaultOptions, getHeightWithUnits } from "./utils";
```

Comment:
Namings in this file are pretty vague and we are exporting them in main index file of the sdk-ui-kit.
I'm not sure if all of them are related only to embed insight dialog or they are more reusable (eg NumericInput, CodeArea etc...). Please reflect this in namings, or namespace these exports in some object.

Reply:
Name of getDefaultOptions and getHeightWithUnits changed